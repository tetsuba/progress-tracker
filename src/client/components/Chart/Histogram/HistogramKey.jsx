import React from 'react'

import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faSquare } from '@fortawesome/free-solid-svg-icons'

export default function HistogramKey() {
  const styles = {
    textAlign: 'center',
    width: '100%',
  }
  return (
    <div className="HistogramKey" style={styles}>
      <span style={{ color: '#309c3f', marginRight: '20px' }}>
        <FontAwesomeIcon icon={faSquare} /> Correct
      </span>
      <span style={{ color: 'red' }}>
        <FontAwesomeIcon icon={faSquare} /> wrong
      </span>
    </div>
  )
}
