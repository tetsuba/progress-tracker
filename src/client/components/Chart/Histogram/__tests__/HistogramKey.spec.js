import { testRenderer } from '../../../../../test/testHelper'
import HistogramKey from '../HistogramKey'

describe('<HistogramKey>', () => {
  describe('@Render', () => {
    it('Should render', () => {
      const wrapper = testRenderer(HistogramKey)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
