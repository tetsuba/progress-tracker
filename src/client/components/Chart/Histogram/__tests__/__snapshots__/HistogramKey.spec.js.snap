// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HistogramKey> @Render Should render 1`] = `
<div
  className="HistogramKey"
  style={
    Object {
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <span
    style={
      Object {
        "color": "#309c3f",
        "marginRight": "20px",
      }
    }
  >
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Object {
          "icon": Array [
            448,
            512,
            Array [],
            "f0c8",
            "M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48z",
          ],
          "iconName": "square",
          "prefix": "fas",
        }
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
     Correct
  </span>
  <span
    style={
      Object {
        "color": "red",
      }
    }
  >
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Object {
          "icon": Array [
            448,
            512,
            Array [],
            "f0c8",
            "M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48z",
          ],
          "iconName": "square",
          "prefix": "fas",
        }
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
     wrong
  </span>
</div>
`;
